Change manually models name to be display

ManojDatt opened the pull request.

f1nality committed:
Remove old imports

f1nality committed:
Add label to all checkboxes without label

f1nality committed:
Fix mistype

f1nality committed:
Fix multiple admin sites support

f1nality committed:
Update documentation

f1nality committed:
Include dashboard locales

f1nality committed:
Fix datetime widget for editable changelist

f1nality committed:
Fix siblings for custom admin site

f1nality committed:
Fix AppList and ModelList models/exclude parsers

f1nality committed:
Fix pinned application user filtering

f1nality committed:
Fix empty branding visibility

f1nality committed:
Fix ie dashboard list items wrapping

f1nality committed:
Improve docs navigation

f1nality committed:
Fix ie sidebar popup items spacing

f1nality committed:
Fix dashboard module height wrong after height animation

f1nality committed:
Update translation contribution docs

f1nality committed:
Fix dashboard module change form breadcrumbs

f1nality committed:
Improve paginator 'show all' layout

f1nality committed:
Build static

f1nality committed:
Add support for filters with multiple select

f1nality committed:
Refactor inputs height

f1nality committed:
Fix select2 list item style

f1nality committed:
Fix select2 arrow padding

f1nality committed:
Make select2 width auto

f1nality committed:
Fix changelist filters multiple selects style

f1nality committed:
Build static

f1nality committed:
Update changelog

f1nality committed:
Update status

f1nality committed:
Update version

f1nality committed:
Merge branch 'dev'

f1nality committed:
Change license to AGPLv3

f1nality committed:
Fix multiple filters behavior

f1nality committed:
Merge branch 'dev'

f1nality committed:
Update setup.py info

f1nality committed:
Merge branch 'dev'

f1nality committed:
Fix missed old info

f1nality committed:
Merge branch 'dev'

f1nality committed:
Remove old info

f1nality committed:
Merge branch 'dev'

f1nality committed:
Add compact inline

f1nality committed:
Refactor

f1nality committed:
Scroll to bottom on inline item adding

f1nality committed:
Add responsiveness to compact inline

f1nality committed:
Fix selector

f1nality committed:
Tweak fade effects

f1nality committed:
Add scroll to top on inline item delete

f1nality committed:
Truncate inline add row

f1nality committed:
Update documentation

f1nality committed:
Build static

f1nality committed:
Update version

f1nality committed:
Update changelog

f1nality committed:
Merge branch 'dev'

f1nality committed:
Add CompactInline class

f1nality committed:
Merge branch 'dev'

f1nality committed:
Update texts

f1nality committed:
Merge branch 'dev'

darccio committed:
Remove RequestFactory

darccio committed:
Added request param to get_model_queryset

darccio committed:
Pass request from context to get_model_queryset

darccio committed:
Update test_tags.py

darccio committed:
Fixed null url on test_jet_sibling_object_previous_url

f1nality committed:
Merge pull request #115 from imdario/master

Remove mock request object from jet.utils.get_model_queryset

mord4z committed:
Dashboard pt-br translation (#119)

Added brazilian portuguese translations for Dashboard

sedir committed:
Support for Brazilian Portuguese localization (#107)

f1nality committed:
Merge branch 'master' into dev

dbartenstein committed:
initial German translation (#109)

SalahAdDin committed:
Localization (#106)

* Rename CONTRIBUTION.rst to CONTRIBUTING.rst

* Now yer, localization files for spanish language.

* Update localization files.

f1nality committed:
Change es LRU label to URL

f1nality committed:
Add text wrapping to breadcrumbs

f1nality committed:
Compile translations

f1nality committed:
Merge branch 'dev'

retailify committed:
Django 1.10 / Update jet_custom_apps_example.py (#121)

retailify committed:
Update jet_custom_apps_example.py (#122)

Commit:
Added czech translation

Commit:
Added czech Google Analytics translation

timur-orudzhov committed:
proper static tag for storages

timur-orudzhov committed:
load proper static tag

f1nality committed:
Remove redundant label

f1nality committed:
Build js

f1nality committed:
Merge branch 'dev'

f1nality committed:
Merge branch 'dev' of github.com:geex-arts/django-jet into dev

f1nality committed:
Fix compactinline opening first navigation item when there are no items

f1nality committed:
Respect inline max forms for compactinline

f1nality committed:
Respect draggable setting

f1nality committed:
Save linklist module draggable/deletable/collapsible settings

f1nality committed:
Merge commit 'b901b996bd439c5828247179762c082cdce68b03' into dev

f1nality committed:
Fix django 1.10 stacked inlines

f1nality committed:
Fixes related popup links for new inline items

f1nality committed:
Fix delete confirmation submit button misplacement

f1nality committed:
Build static

f1nality committed:
Merge branch 'dev'

f1nality committed:
Update changelog

f1nality committed:
Update version

f1nality committed:
Merge branch 'dev'

f1nality committed:
Fixed wrong if

f1nality committed:
Merge branch 'dev'

dbartenstein committed:
Improved German translation (#129)

* Better German translation of 'valid'.

* Added compiled msg file

timur-orudzhov committed:
add date-range filter

timur-orudzhov committed:
changes files after buid

timur-orudzhov committed:
add package django_form_admin

timur-orudzhov committed:
remove formadmin package

timur-orudzhov committed:
render form properly

f1nality committed:
Fix related popup links for tabular inline

michaelkuty committed:
Fix encoding error in jet_popup_response_data. (#138)

UnicodeEncodeError: 'ascii' codec can't encode character u'\xed' in position 13: ordinal not in range(128)
(5 additional frame(s) were not displayed)
...
  File "jet/templatetags/jet_tags.py", line 270, in jet_popup_response_data
    'obj': str(context.get('obj')),

Commit:
use HttpRequest().resolver_match instead of resolve() (#131)

f1nality committed:
Merge branch 'dev' of github.com:geex-arts/django-jet into dev

f1nality committed:
Add dashboard views tests

f1nality committed:
Fix gulp tasks chaining

f1nality committed:
Fix RelatedFieldAjaxListFilter attributes to template passing

michaelkuty committed:
Add change message as tooltip to recent action dashboard module.

timur-orudzhov committed:
add date-range-filter as a thirdparty package

f1nality committed:
Specify google dependency version

f1nality committed:
Disable breadcrumbs text wrapping for mobile

f1nality committed:
Add libi.io url

f1nality committed:
Build static

f1nality committed:
Merge branch 'dev'

timur-orudzhov committed:
add daterange filter to jet

timur-orudzhov committed:
bug fix: wrong admin date widget when django-suit is installed

timur-orudzhov committed:
add comments

normal-cock committed:
fix that account with just adding permission couldn't open index page of admin system

f1nality committed:
Merge branch 'date_range_filter' into from-date_range_filter

# Conflicts:
#	gulpfile.js
#	jet/filters.py
#	jet/static/jet/css/themes/default/base.css
#	jet/static/jet/css/themes/default/base.css.map
#	jet/static/jet/css/themes/green/base.css
#	jet/static/jet/css/themes/green/base.css.map
#	jet/static/jet/css/themes/light-blue/base.css
#	jet/static/jet/css/themes/light-blue/base.css.map
#	jet/static/jet/css/themes/light-gray/base.css
#	jet/static/jet/css/themes/light-gray/base.css.map
#	jet/static/jet/css/themes/light-green/base.css
#	jet/static/jet/css/themes/light-green/base.css.map
#	jet/static/jet/css/themes/light-violet/base.css
#	jet/static/jet/css/themes/light-violet/base.css.map
#	jet/static/jet/js/build/bundle.min.js

f1nality committed:
Update submit button style

f1nality committed:
Merge branch 'from-date_range_filter' into dev

f1nality committed:
fix that account with just adding permission couldn't open index page of admin system (#144)

f1nality committed:
Fix rangefilter inputs style

michaelkuty committed:
Add change message as tooltip to recent action dashboard module.

michaelkuty committed:
Fix rebase.

f1nality committed:
Merge branch 'feature/tooltips' of github.com:michaelkuty/django-jet into dev

# Conflicts:
#	jet/dashboard/templatetags/jet_dashboard_tags.py

f1nality committed:
Build css

f1nality committed:
Add singlings urls resolving checks

f1nality committed:
Update documentation

f1nality committed:
Update changelog

f1nality committed:
Update version

f1nality committed:
Update libi.io url

f1nality committed:
Fix select class name override

f1nality committed:
Fix m2m ajax filters

f1nality committed:
Build js

f1nality committed:
Add permissions check to model lookup view

f1nality committed:
Add bookmarks staff check

f1nality committed:
Add pinned apps staff check

f1nality committed:
Add dashboard views is_staff checks

f1nality committed:
Update version, changelog

f1nality committed:
Add user validation for load_dashboard_module_view

f1nality committed:
Add is_staff check

Bashar committed:
Arabic translation by KUWAITNET (#178)

* Adding Arabic translation

* Adding Arabic Translation

* Adding js Arabic Translation

* Dashboard's Arabic translation file

hbiboluo committed:
add: zh-cn language support. (#169)

lburdzy committed:
Added Polish translation (#172)

michaelkuty committed:
Add fallback to window.opener which supports old django popups. (#167)

brenouchoa committed:
fix permission error on ModelLookupForm (#174)

f1nality committed:
Fix non existant object

f1nality committed:
Remove powered by django jet label

f1nality committed:
Build static

f1nality committed:
Update changelog

f1nality committed:
Update version

SalahAdDin commented:
Hey, very extensive. 👍

f1nality committed:
Fix inline-group-row:added event

f1nality committed:
Fix datetime init after inline row added

f1nality committed:
Build js

grigory51 committed:
pin sidebar and save state in cookie

grigory51 committed:
build

grigory51 committed:
fix z-index in popup

grigory51 committed:
build

f1nality committed:
Add new menu customization

f1nality committed:
Merge branch 'feature/hide-pin-menu' into from-hide-pin-menu

f1nality committed:
Merge branch 'from-hide-pin-menu' into dev

f1nality committed:
Build static

f1nality committed:
Update sibling buttons

grigory51 committed:
Remove burger from login page (#192)

* remove breadcrumbs totaly from login page

* style

* build

grigory51 committed:
Fix merge (#193)

* remove breadcrumbs totaly from login page

* style

* build

* fix merge

* build

* fix padding

f1nality committed:
Fix django.jQuery select change events

f1nality committed:
Fix app url

f1nality committed:
Allow app based model names

f1nality committed:
Fix app settings overriding

f1nality committed:
Merge branch 'dev' of github.com:geex-arts/django-jet into dev

f1nality committed:
Update breadcrums layout

f1nality committed:
Refactor selectors

f1nality committed:
Change none label

f1nality committed:
Add url check

f1nality committed:
Fix branding paddings

f1nality committed:
Build static

f1nality committed:
Fix translation

f1nality committed:
Update changeform style

f1nality committed:
Update select disabled style

f1nality committed:
Fix unicode

f1nality committed:
Build static

f1nality committed:
Fix django 1.11 context issue

f1nality committed:
Fix initial value for select2 ajax fields when post

f1nality committed:
Implement flatten as fallback

f1nality committed:
Update testing django versions

f1nality committed:
Add quotes

f1nality committed:
Fix django version

f1nality committed:
Fix import

f1nality committed:
Update exclude

f1nality committed:
Fix none object

f1nality committed:
Fix func name

f1nality committed:
Fix tests

f1nality committed:
Add quotes

f1nality committed:
Fix exclude

f1nality committed:
Add python 2.6 depency

f1nality committed:
Change sidebar pinned default

f1nality committed:
Fix modules context

f1nality committed:
Add sidebar check

f1nality committed:
Reverse

f1nality committed:
Fix login button

f1nality committed:
Build static

alin23 committed:
Fix urlresolvers deprecation in Django 1.11 (#210)

f1nality committed:
Update documentation

f1nality committed:
Merge branch 'dev' of github.com:geex-arts/django-jet into dev

f1nality committed:
Rename models key to items

f1nality committed:
Update documentation

f1nality committed:
Add python 3.5, 3.6

f1nality committed:
Fix default menu items

f1nality committed:
Fix pip

f1nality committed:
Fix exclude

f1nality committed:
Update version, changelog

f1nality committed:
Fix default menu items #2

pyzenberg committed:
Added Persian/Farsi translations

all-4-1-1-4-all commented:
Please come to my Discord server so we can organize if you like:

* https://discord.gg/3yr5yeH

Welcome! 😄

HarryLafranc committed:
Fixes for Django 2.0+

leonardoarroyo committed:
Fix locale names

HarryLafranc committed:
Another fixes for Django 2.0

HarryLafranc committed:
FIX: Using User.is_authenticated() and User.is_anonymous() as methods rather than properties is no longer supported.

HarryLafranc committed:
Don’t need packages as Django takes self.packages

f1nality committed:
Merge branch 'feature/fa-translation' into dev

f1nality committed:
Merge branch 'dev' of github.com:HarryLafranc/django-jet into feature/django-2.0

f1nality committed:
Fix tags

f1nality committed:
Update travis

f1nality committed:
Fix url include

f1nality committed:
Fix on delete

f1nality committed:
Fix locale

f1nality committed:
Fix is authenticated

f1nality committed:
Fix height

f1nality committed:
Build static

f1nality committed:
Fix urls

f1nality committed:
Fix middlewares

f1nality committed:
Merge branch 'feature/django-2.0' into dev

f1nality committed:
Merge branch 'dev' of github.com:leonardoarroyo/django-jet into dev

f1nality committed:
Updated changelog

f1nality committed:
Update version

f1nality committed:
Update .travis.yml

woutor committed:
Fix google analytics widget (#301)

* Make _lock attribute available

* Allow server-side token refresh

f1nality committed:
Update README.rst

f1nality committed:
Update README.rst

SalahAdDin commented:
@f1nality

rense committed:
Django 2.1 compatability fix (#345)

Skips adding 'sortable_by' arg for Django versions < 2.1

lmwgv committed:
Remove duplicate entries in autocomplete (#356)

When "autocomplete_search_fields" contains a ManyToMany related field, the resulting queryset has duplicates. Adding a distinct() removes them.

danielquinn committed:
Add an `s` for grammar. (#311)

* Update .travis.yml (#282)

* Add an `s` for grammar.

jpic committed:
Add French locale (#309)

tkanemoto committed:
Fix get_model_queryset exception when model_admin is None (#337)

f1nality committed:
Update changelog

f1nality committed:
Update README.rst

f1nality committed:
Update README.rst

f1nality committed:
Create codeql.yml
